framework/include/base/FEProblem.h

```diff
@@ -1022,6 +1046,12 @@ class FEProblem :
   /// Preconditioner description
   std::string _pc_description;
 
+  /// Whether or not an exception has occurred
+  bool _has_exception;
```

Comment:
These should both really be private.  Not a big deal... but we do need to get better about making things private (Yes, @permcody , I'm agreeing with you!)